### PR TITLE
Fix: add a check on generated files

### PR DESCRIPTION
**Pull request by eloycoto**
Signed-off-by: Eloy Coto <eloy.coto@acalustra.com>

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -21,8 +21,8 @@ jobs:
     - name: Test
       run: make test
 
-    - name: Coverage
-      run: make test-create-coverage
+    - name: Checking generated files are up to date
+      run: git diff --quiet internal/ client/ restapi/ || echo "There are changes no committed"; git diff internal/ client/ restapi/
```
Comment (jakub-dzon) [inline .github/workflows/go.yml:25]: "There are changes no committed" -> "There are not committed changes"

**Comment by jakub-dzon** [comment]
The change looks good, but now it requires the code to be regenerated...

**Comment by masayag** [comment]
> The change looks good, but now it requires the code to be regenerated...

That will be fixed by merging https://github.com/jakub-dzon/k4e-operator/pull/48

**Comment by eloycoto** [comment]
Ok, it was still WIP. Now should be ok to merge if #48 fix the generation.

**Review by pkliczewski — COMMENTED** [review]
/lgtm

**Review by masayag — APPROVED** [review]

**Pull request merged**